### ISSUE TITLE
please publish the source code for Emacs+

### ISSUE THREAD

**Issue by tbcs** [MulgaSoft/intellemacs#8]
I just reported an issue (#7) which I would have gladly looked into, it looks like it might be easily fixed, but I can't find the sources to the project anywhere.  The [MulgaSoft website](http://www.mulgasoft.com/intellemacsplus/mpl-v-1-0) states that Emacs+ for IntellliJ is licensed under MPL.  Is the repository somewhere else?

**Comment by rdgoite** [comment]
Yeah, I think I could also help if you would kindly make the source code available.
